migrations/versions/0208_remove_monthly_billing_.py

```diff
@@ -0,0 +1,38 @@
+"""
```

Comment:
this shouldto go into a separate PR. Migrations run before code change, so we need to remove all code references to the table before we can delete it.

Reply:
Ahh, good point, I didn't think about it. I will move the migration file into a separate PR 👍